orbita.php

```diff
@@ -11,7 +11,7 @@
  * Plugin Name:     Órbita
  * Plugin URI:      https://gnun.es
  * Description:     Órbita é o plugin para criar um sistema Hacker News-like para o Manual do Usuário
- * Version:         1.10.4
+ * Version:         1.10.4.3
```

Comment:
```suggestion
 * Version:         1.10.5
```

Acho que o `.3` ali no final é só em caso de correção de bugs